### ISSUE TITLE
Filemarker requirements?

### ISSUE THREAD

**Issue by thinkmorebetter** [ginjo/rfm#13]
Hi Bill,

Couldn't find any other way to contact you other than this way, so apologies for the "issue"

I'm completely new for Filemaker, but I was recently asked if it was possible to integrate a Filemaker database at a friend's office with a cloud-based website built in Rails. Clearly, this gem looks like it'll work for this, but I'm just wondering if there are any additional requirements that I'm not yet aware of.

For example, there's Filemaker Pro and then there's Server. Would this gem be able to connect with just the Pro version?

Thanks,
Alex

**Comment by kjohnston** [comment]
Does FileMaker 13 and/or 14 use the XML API with the fmresultset XML?

**Comment by ginjo** [comment]
Yes and Probably. Filemaker Server 13 serves up the fmresultset grammar and XML just fine. I'm guessing Filemaker Server 14 does as well, but I haven't used it yet.

**Comment by kjohnston** [comment]
@ginjo, thanks for the prompt response and for your continued support of this project.

**Comment by kjohnston** [comment]
We've been running 3.0.9 against FileMaker 14 for the last couple of weeks and all has been well.  Thanks for the continued support!

**Comment by kjohnston** [comment]
@ginjo we've been using 3.0.11 against FileMaker 15 for a while now and all is well.  Any plans to support FileMaker >= 16?

**Comment by ginjo** [comment]
@kjohnston , yes absolutely 😃.  I've just recently acquired access to a FM Server 16 for testing, so I'll check it out. However, as far as I know, there are no changes to the XML interface in FM Server 16. So you should be able to use Rfm with FM Server 16 right now. Of course, if you do find bugs, feel free to post them as issues.

Re the new FM API feature, I'm mulling over whether it makes any sense to incorporate that into Rfm. I don't think there is anything it can do that the XML interface doesn't support. I suppose if someone wants raw JSON responses from FM Server in the context of a ruby application, that might be something worth considering. This is probably a good point to create a new issue/topic for discussion on here (github) or on the Rfm Community google-group.

**Comment by Willem-Jan** [comment]
HI There,
I have written a wrapper around the data api. My first project with ruby...
It is a private git. Would anyone like to join and make the project better ?

**Comment by kjohnston** [comment]
@ginjo awesome, thanks!
